### DISCUSSION TITLE
Status Offline to XP-30 via Novation SL Mk3

### DISCUSSION THREAD

**Discussion by lattetown** [RomanKubiak/ctrlr#349] [Q&A] [answered]
I'm experiencing and issue connecting the editor to my XP-30 through my usb connected Novation SL MK3 midi keyboard.  I've confirmed the XP-30 settings match the documentation:

SYSTEM → MIDI → RECEIVE MIDI → Bank Select = on
SYSTEM → MIDI → SYS-EXC → Device # = 17
SYSTEM → MIDI → SYS-EXC → Rx. Exc = on

I have a 5-pin midi cable going OUT of the SL Mk3 and into the XP-30, and the controller is playing notes manually pressed on the Novation.  However the status remains offline with any of these settings:

Input/Output device: Novation SL MkIII MIDI  (All Channels, Channel 1, Channel 2)
Inout/Output device: Novation SL MkIII InControl (All Channels, Channel 1, Channel 2

![Screen Shot 2021-08-16 at 1 17 21 PM](https://user-images.githubusercontent.com/25696235/129624388-03884124-243e-45bd-abbe-d1e7d191fbb7.png)
)

**Comment by Ted-juh** [comment]
Does the novation sl mkIII support Sysex? Might be what you need to get full functionality besides midi CC.

**Reply by lattetown** [reply]
It turns out the Novation SL MK3 has multiple input/output devices, and the "midi" devices in the screenshot above are actually for midi via USB. Instead it should use the devices labeled "DIN" like this...

![Screen Shot 2021-08-17 at 8 51 25 PM](https://user-images.githubusercontent.com/25696235/129834312-1a31c2d8-5ad8-493b-bcf6-8b02b0e3f3d5.png)

**Comment by lattetown** [comment, marked as answer]
Use the input and output devices labeled: **_Novation SL MkIII to DIN 1_**
